GPU/GPUCommon.cpp

```diff
@@ -3080,9 +3080,9 @@ bool GPUCommon::PerformMemoryCopy(u32 dest, u32 src, int size, GPUCopyFlag flags
 	// Track stray copies of a framebuffer in RAM. MotoGP does this.
 	if (framebufferManager_->MayIntersectFramebuffer(src) || framebufferManager_->MayIntersectFramebuffer(dest)) {
 		if (!framebufferManager_->NotifyFramebufferCopy(src, dest, size, flags, gstate_c.skipDrawReason)) {
-			// We use a little hack for PerformReadbackToMemory/PerformWriteColorFromMemory using a VRAM mirror.
+			// We use a matching values in PerformReadbackToMemory/PerformWriteColorFromMemory.
 			// Since they're identical we don't need to copy.
-			if (!Memory::IsVRAMAddress(dest) || (dest ^ 0x00400000) != src) {
+			if (dest != src) {
```

Comment:
we don't need to mask any bits in the check?

Reply:
I mean, we could, but it wasn't before - I'm just trying to skip the common case of a detected `PerformWriteColorFromMemory()` which will pass the same value as dest and src, but previously xored one of the values.  Since we no longer xor, we can skip the VRAM check and directly check equality.

-[Unknown]